static/js/zcommand.js

```diff
@@ -87,6 +88,27 @@ exports.process = function (message_content) {
         return true;
     }
 
+    if (tokens[0] === '/subscribe') {
+        tokens[0] = tokens[0].replace('/', '');
```

Comment:
You could say `tokens[0] = 'subscribe'` here to be more direct.  It's possible we should stop stripping the leading slash.  I'll ask on chat.